sorting/java/BubbleSort.java

```diff
@@ -0,0 +1,27 @@
+

```

Comment:
Rename file's name from `BubbleSort` to `bubble-sort`

Reply:
done